sdk/attestation/azure-security-attestation/NOTICE.txt

```diff
@@ -0,0 +1,32 @@
+azure-security-attestation
+
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties. Microsoft makes certain
+open source code available at https://3rdpartysource.microsoft.com, or you may
+send a check or money order for US $5.00, including the product name, the open
+source component name, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the
+extent required to debug changes to any libraries licensed under the GNU Lesser
+General Public License.
+
+------------------------------------------------------------------------------
+
+Azure SDK for C++ uses third-party libraries or other resources that may be
```

Comment:
Do we have any third party library dependencies we are including to add to the notice?

For example, other services have a few other things:
https://github.com/Azure/azure-sdk-for-cpp/blob/771b7382c2382c72567af959794c9999479f760d/sdk/storage/azure-storage-blobs/NOTICE.txt#L34

Reply:
Not to my knowledge. OpenSSL should already be included in the 3rd party library dependencies because it's used on Linux.